crates/wadm-types/Cargo.toml

```diff
@@ -1,24 +1,26 @@
 [package]
 name = "wadm-types"
 description = "Types and validators for the wadm API"
-version = "0.7.1"
+version = "0.7.2"
 edition = "2021"
 authors = ["wasmCloud Team"]
 keywords = ["webassembly", "wasmcloud", "wadm"]
 license = "Apache-2.0"
 repository = "https://github.com/wasmcloud/wadm"
 
 [features]
-default = []
-wit = ["wit-bindgen-wrpc"]
+default = ["std"]
+std = ["dep:tokio"]
+wasm = []
```

Comment:
Ahh ok when I originally got this working, I depended on this being wit_bindgen types generated in wasm. The reason I added the feature is because the current bindgen for WIT uses wrpc_bindgen. I managed to get this working without it, aslong as i set default-features = false to remove the std.

Let me remove wasm flag